src/main/kotlin/com/cognifide/gradle/aem/instance/tasks/InstanceWorkflow.kt

```diff
@@ -0,0 +1,55 @@
+package com.cognifide.gradle.aem.instance.tasks
+
+import com.cognifide.gradle.aem.common.instance.names
+import com.cognifide.gradle.aem.common.instance.service.repository.Node
+import com.cognifide.gradle.aem.common.instance.service.repository.ResourceType
+import com.cognifide.gradle.aem.common.tasks.Instance
+import org.gradle.api.tasks.Internal
+import org.gradle.api.tasks.TaskAction
+
+open class InstanceWorkflow : Instance() {
+
+    private val notifier = common.notifier
+
+    @Internal
+    val model = aem.obj.string {
+        convention("update_asset")
+        common.prop.string("instance.workflow")?.let { set(it) }
+    }
+
+    @Internal
+    val path = aem.obj.string {
+        convention(Node.DAM_PATH)
+        common.prop.string("instance.workflow.path")?.let { set(it) }
+    }
+
+    @Internal
+    val resourceType = aem.obj.string {
+        convention(ResourceType.ASSET.value)
+        common.prop.string("instance.workflow.resourceType")?.let { set(it) }
+    }
```

Comment:
What about `aem.obj.typed<ResourceType>{..}`?

Reply:
I am providing usually own of( method which is case insensitive